<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/image/editor.svg</normaloff>:/icon/image/editor.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>font</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_underline">
        <property name="text">
         <string>underline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_italic">
        <property name="text">
         <string>italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_bold">
        <property name="text">
         <string>bold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>color</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_red">
        <property name="text">
         <string>red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_black">
        <property name="text">
         <string>black</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_blue">
        <property name="text">
         <string>blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="plainText">
      <string>this is dialog</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
